components/com_kunena/site/template/crypsis/layouts/category/item/default.php

```diff
@@ -38,106 +38,112 @@
 			->set('display', true); ?>
 	</div>
 
-<form action="<?php echo KunenaRoute::_('index.php?option=com_kunena'); ?>" method="post">
-	<input type="hidden" name="view" value="topics"/>
-	<?php echo JHtml::_('form.token'); ?>
-	<div class="kbuttons">
-		<ul class="inline">
-			<?php if ($categoryActions) : ?>
-				<li class="hidden-phone">
-					<?php echo implode($categoryActions); ?>
-				</li>
-			<?php endif; ?>
-		</ul>
-	</div>
-	<table class="table table-bordered">
-		<thead>
-		<tr>
-			<td class="span1 center hidden-phone">
-				<a id="forumtop"> </a>
-				<a href="#forumbottom">
-					<i class="icon-arrow-down hasTooltip"></i>
-				</a>
-			</td>
-			<td class="span<?php echo $cols ?>">
-				<?php echo JText::_('COM_KUNENA_GEN_SUBJECT'); ?>
-			</td>
-			<td class="span2 hidden-phone">
-				<?php echo JText::_('COM_KUNENA_GEN_REPLIES'); ?> / <?php echo JText::_('COM_KUNENA_GEN_HITS'); ?>
-			</td>
-			<td class="span3">
-				<?php echo JText::_('COM_KUNENA_GEN_LAST_POST'); ?>
-			</td>
-			<?php if (!empty($this->topicActions)) : ?>
-				<td class="span1 center">
-					<label>
-						<input class="kcheckall" type="checkbox" name="toggle" value=""/>
-					</label>
-				</td>
-			<?php endif; ?>
-		</tr>
-		</thead>
-		<?php endif; ?>
-		<?php if (empty($this->topics)) : ?>
-			<div class="alert">
-				<?php echo JText::_('COM_KUNENA_VIEW_NO_TOPICS') ?>
-			</div>
-
-		<?php else :
-
-		/** @var KunenaForumTopic $previous */
-		$previous = null;
-
-		foreach ($this->topics as $position => $topic) {
-			echo $this->subLayout('Topic/Row')
-				->set('topic', $topic)
-				->set('spacing', $previous && $previous->ordering != $topic->ordering)
-				->set('position', 'kunena_topic_' . $position)
-				->set('checkbox', !empty($this->topicActions))
-				->setLayout('category');
-			$previous = $topic;
-		}
-
-		?>
-		<tfoot>
-		<tr>
-			<td class="center hidden-phone">
-				<a id="forumbottom"> </a>
-				<a href="#forumtop" rel="nofollow">
-					<span class="divider"></span>
-					<i class="icon-arrow-up hasTooltip"></i>
-				</a>
-				<?php // FIXME: $this->displayCategoryActions() ?>
-			</td>
-			<td colspan="6" class="hidden-phone">
-				<div class="input-append">
-
-					<?php if (!empty($this->moreUri)) echo JHtml::_('kunenaforum.link', $this->moreUri,
-						JText::_('COM_KUNENA_MORE'), null, null, 'follow'); ?>
-
+	<form action="<?php echo KunenaRoute::_('index.php?option=com_kunena'); ?>" method="post">
+		<input type="hidden" name="view" value="topics"/>
+		<?php echo JHtml::_('form.token'); ?>
+		<div class="kbuttons">
+			<ul class="inline">
+				<?php if ($categoryActions) : ?>
+					<li class="hidden-phone">
+						<?php echo implode($categoryActions); ?>
+					</li>
+				<?php endif; ?>
+			</ul>
+		</div>
+		<table class="table table-bordered">
+			<?php if (!empty($this->topics)) : ?>
```

Comment:
i have also fixed some other issue's on this file, i i revert it then all is gone, and i don't know what i fixed there. The best is to merge this pr, and then change it to your filters.